crates/ruff_python_formatter/src/expression/mod.rs

```diff
@@ -407,9 +406,12 @@ impl<'input> CanOmitOptionalParenthesesVisitor<'input> {
                 attr: _,
                 ctx: _,
             }) => {
+                self.visit_expr(value);
```

Comment:
Can you test if we need the same for `Subscript`? Because it has the same pattern or is it not required because we never want to break subscripts?

Reply:
I did look into this but perhaps I didn't understand what you're referencing. Are you referring to a subscript with a trailing attribute, like `foo[bar].baz`? Or a subscript following a function call, like `foo(bar)[baz]`? I believe I added test coverage for both, and both are consistent with Black.

Reply:
Black output:

```python
ct_match = (
    self.get_content_type["obj=rel_obj, using=instance._state.db"] == aaaaaaaaaaact_id
)

ct_match = (
    aaaaaaaaaaact_id == self.get_content_type["obj=rel_obj, using=instance._state.db"]
)

```

So black prefers not to break subscripts. Can we remove the `Subscript` case from `has_own_parentheses`?

Reply:
Will investigate, thank you for clarifying.